components/constants/pref_names.h

```diff
@@ -51,7 +51,6 @@ inline constexpr char kReferralAttemptTimestamp[] =
     "brave.referral.referral_attempt_timestamp";
 inline constexpr char kReferralAttemptCount[] =
     "brave.referral.referral_attempt_count";
-inline constexpr char kReferralHeaders[] = "brave.referral.headers";
```

Comment:
It would be great to have clean up code for this deprecated local state in `MigrateObsoleteLocalStatePrefs(PrefService* local_state)`

Reply:
Good point. That should have been done 2.5 years ago as part of https://github.com/brave/brave-core/pull/9157 when the dynamic list moved to a hard-coded one.

I'll make a note to follow up on this.